testdata/templates/peer.yaml

```diff
@@ -0,0 +1,25 @@
+service: foo
+procedure: Simple::foo
+peer: 127.0.0.1:8080
+peers:
+- 127.0.0.1:8080
+- 127.0.0.1:8081
+peerList: peers.json
+caller: bar
+shardKey: sk
+routingKey: rk
+routingDelegate: rd
+thrift: testdata/templates/foo.thrift
```

Comment:
similar to `PeerList`, should this location be relative to the location of the yaml template (so this should become `foo.thrift`)